airflow-core/src/airflow/ui/src/i18n/check_translations_completness.py

```diff
@@ -0,0 +1,154 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#   "rich",
+# ]
+# ///
+from __future__ import annotations
+
+import json
+from collections import defaultdict
+from pathlib import Path
+from typing import Any
+
+from rich import print
+from rich.console import Console
+from rich.panel import Panel
+from rich.table import Table
+
+LOCALES_DIR = Path(__file__).parent / "locales"
+
+
+def get_locale_files() -> dict[str, list[str]]:
+    locales: dict[str, list[str]] = {}
+    for locale_dir in LOCALES_DIR.iterdir():
+        if locale_dir.is_dir():
+            files = [f.name for f in locale_dir.iterdir() if f.suffix == ".json"]
+            locales[locale_dir.name] = files
+    return locales
```

Comment:
```suggestion
    retrun {
        locale_dir.name: [f.name for f in locale_dir.iterdir() if f.suffix == ".json"]
        for locale_dir in LOCALES_DIR.iterdir()
        if locale_dir.is_dir()
    }
```

probably not something really matters but it gives me a better sense on what the key, value are

Reply:
Why not.